std/bitmanip.d

```diff
@@ -107,7 +107,7 @@ private template createAccessors(
 
         static if (is(T == bool))
         {
-            static assert(len == 1);
+            static assert(len == 1, "'"~name~"' definition problem: type 'bool' is allowed only for single-bit fields");
```

Comment:
spaces around `~`

Reply:
fixed, but in fact, this file has complete anarchy.

Reply:
Yes we try to gradually create consistency whenever it comes up. Though there's the idea of using dfmt to format the git diff automatically.